src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const TMP_EXTRACT_DIR = './tmp/h5p/';
+
+export const MAX_FILE_SIZE = 1024 * 1024 * 50; // 50 MB
+export const MAX_NON_FILE_FIELDS = 0;
+export const MAX_FILES = 1;
+
+export const H5P_ITEM_TYPE = 'h5p';
```

Comment:
What we could do (later) is have those types in `graasp-utils` and refer it from there.